apis/v1alpha2/converter/httproute.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 The Kubernetes Authors.
+Copyright 2022 The Kubernetes Authors.
```

Comment:
This feels a little weird in that this seems somewhere between a file being moved (stick with original year) and a net new file (use current year). I think I'd lean towards sticking with the original year here, but don't feel strongly.

Reply:
I had the same exact thinking and was split between wanting to keep the year the original content was added or going with the year the file was added 🤔

I think going with the year the content was added best reflects that this is a move and not an original. Will change.